addons/pos_sale/models/crm_team.py

```diff
@@ -0,0 +1,96 @@
+            amount_total = 0
+            for pos_config in team.pos_config_ids:
+                for session in pos_config.session_ids.filtered(lambda s: s.state == 'opened'):
+                    amount_total += sum(session.mapped('order_ids.amount_total'))
+            team.pos_order_amount_total = amount_total
+
+    @api.model
+    def _verify_team_type(self, values):
+        if values.get('team_type') == 'pos':
+            values.update({'use_quotations': False, 'use_invoices': False})
+            # check if crm is installed
+            if hasattr(self, 'use_opportunities'):
+                values.update({'use_opportunities': False, 'use_leads': False, 'dashboard_graph_model': 'sales'})
+        return super(CrmTeam, self)._verify_team_type(values)
+
+    def _graph_data(self, start_date, end_date):
+        if self.team_type == 'pos':
+            result = []
+            if self.dashboard_graph_group == 'user':
+                order_data = self.env['pos.order'].read_group(
+                    domain=[
+                        ('create_date', '>=', fields.Date.to_string(start_date)),
+                        ('create_date', '<=', fields.Datetime.to_string(datetime.combine(end_date, datetime.max.time()))),
+                        ('config_id', 'in', self.pos_config_ids.ids),
+                        ('state', 'in', ['paid', 'done', 'invoiced'])],
+                    fields=['user_id', 'amount_total'],
+                    groupby=['user_id']
+                )
+                for data_point in order_data:
+                    result.append({'x_value': data_point.get('user_id')[0], 'y_value': data_point.get('amount_total')})
+            else:
+                order_data = self.env['pos.order'].read_group(
+                    domain=[
+                        ('create_date', '>=', fields.Date.to_string(start_date)),
+                        ('create_date', '<=', fields.Datetime.to_string(datetime.combine(end_date, datetime.max.time()))),
+                        ('config_id', 'in', self.pos_config_ids.ids),
+                        ('state', 'in', ['paid', 'done', 'invoiced'])],
+                    fields=['create_date', 'amount_total'],
+                    groupby=['create_date:' + self.dashboard_graph_group]
+                )
+                if self.dashboard_graph_group == 'day':
+                    for data_point in order_data:
+                        result.append({'x_value': fields.Date.to_string((fields.datetime.strptime(data_point.get('create_date:day'), "%d %b %Y"))), 'y_value': data_point.get('amount_total')})
+                if self.dashboard_graph_group == 'week':
+                    for data_point in order_data:
+                        # non-standard groupby formatting requires garbage formatting here, hope this'll hold ...
+                        result.append({'x_value': int(data_point.get('create_date:week')[1:3]) - 1, 'y_value': data_point.get('amount_total')})
```

Comment:
To check with framework team if we could change read group result to be directly interpretated by datetime.